assets/scss/components/elements/blog/_single.scss

```diff
@@ -123,6 +127,8 @@
 
 	.container {
 		display: flex;
+		justify-content: var(--justify, center);
```

Comment:
There's an option in customizer where you can remove the container of the cover title. I think you should add `justify-content: var(--justify, center);` on `.nv-post-cover` to cover that case too.